xbmc/interfaces/legacy/Control.h

```diff
@@ -1496,7 +1497,8 @@ namespace XBMCAddon
       ControlSlider(long x, long y, long width, long height, 
                     const char* textureback = NULL, 
                     const char* texture = NULL,
-                    const char* texturefocus = NULL);
+                    const char* texturefocus = NULL,
+                    long orientation = (VERTICAL | HORIZONTAL));
```

Comment:
Not sure what you're trying to do here. This is the same as 0|1 which is the same as 1 and VERTICAL. And it can be int, no need for long;)